OpenRA.Mods.Common/Traits/World/Selection.cs

```diff
@@ -188,7 +188,7 @@ public void DoControlGroup(World world, WorldRenderer worldRenderer, int group,
 				return;
 			}
 
-			var groupActors = controlGroups[group].Where(a => !a.IsDead);
+			var groupActors = controlGroups[group].Where(a => !a.IsDead && a.IsInWorld);
```

Comment:
This can likely simplify to just `a => a.IsInWorld` since dead actors are not in the world either.